src/components/views/Profile/Profile.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { config } from 'dotenv';
+import PropTypes from 'prop-types';
+import Header from '@components/commons/Header';
+import Footer from '@components/commons/utilities/Footer';
+import ProfileImage from './ProfileImage';
+import ProfileDisplay from './ProfileDisplay';
+import './Profile.scss';
+
+config();
+class Profile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tabs: [ 'Followers', 'Articles', 'Bookmarks' ],
+      clickedTab: 'Articles',
+    }
+
+  }
+  render() {
+    const { user, profile, isAuthenticated } = this.props;
+    const {tabs, clickedTab} = this.state;
```

Comment:
Why was this not flagged by your local eslint?

Reply:
I guess its properly destructured.